app/javascript/components/users/SignInComponent.tsx

```diff
@@ -57,47 +78,185 @@ function SignInComponent(props:SignInComponentProps) : JSX.Element {
+				try {
+					await signIn(values);
+				}
+				catch (e: unknown) {
+					// NOTE: We're just swallowing the exception here for now. Might we need to do
+					// something different? Don't know!
+				}
+				finally {
+					formikHelpers.setSubmitting(false);
+				}
 			}
-			finally {
-				formikHelpers.setSubmitting(false);
-			}
-		}
-		}>{(props) => {
+				//Props
+			}>{({ errors, isValid, touched, handleChange }) => {
 				useEffect(() => {
-					setIsValid(props.isValid);
-				}, [props.isValid]);
-
-				return <Form>
-					{/* NOTE: if a Button should submit a form, mark it as type="submit". Otherwise pressing Enter won't submit form*/}
-					<Button data-testid="signInButton" type="submit"
-						variant={'contained'}
-						color={'primary'}>{formatMessage({id: 'hello'})}</Button>
-
-					{componentState === 'submitting' ? "" : <>
-						<div data-testid="signInErrorDiv">{ failed ? lastError.data.error.map((i) => i).join('; ') : ""}</div>
-						<div data-testid="currentUserDiv">{currentUser ? currentUser.id : ""}</div>
-					</>
-					}
-
-				</Form>;
-			}}
+					setIsValid(isValid);
+				}, [isValid]);
+
+				//Form
+				return (
+					<Form>
+						{/* NOTE: if a Button should submit a form, mark it as type="submit". Otherwise pressing Enter won't submit form*/}
 
+						{/* Container of the sign-in component */}
+						<Paper className={classes.paper} elevation={6}>
+							<Grid container wrap="nowrap" spacing={2}>
+								<Grid item
+									direction="column"
+									alignItems="center"
+									justify="center"
+								>
+									{/* Display HoudiniLogo */}
+									<CardMedia component="img"
+										src={logo}
+										title="Houdini"
+									/>
+									{/* Display Login title */}
+									<Box p={3} display="flex" justifyContent="center" alignItems="center">
+										<Typography gutterBottom variant="h5" component="h2">
+											Login
+										</Typography>
+									</Box>
+									<Box display="flex" justifyContent="center" alignItems="center">
+										<Box p={1.5}>
+											<Field component={TextField} name="email" type="text"
+												label={emailLabel}
+												InputProps={{
+													startAdornment: (
+														<InputAdornment position="start">
+															<AccountCircle fontSize="small" />
+														</InputAdornment>
+													),
+												}}
+											/>
+										</Box>
+									</Box>
+									<Box display="flex" justifyContent="center" alignItems="center">
+										<Box p={1.5}>
+											<Field component={TextField} name="password" type="password"
+												label={passwordLabel}
+												InputProps={{
+													startAdornment: (
+														<InputAdornment position="start">
+															<LockOpenIcon fontSize="small" />
+														</InputAdornment>
+													),
+												}} />
+										</Box>
+									</Box>
+                    <Box display="flex" justifyContent="center" alignItems="center">
```

Comment:
I'm not sure I understand what this adds.

Reply:
It helps everything nicely centered.